automation/taskcluster/schedule_snapshot_release.py

```diff
@@ -53,7 +53,7 @@ def as_slugid(name):
         'as_slugid': as_slugid,
         'event': {
             'repository': {
-                'clone_url': params['repository_github_http_url']
+                'html_url': params['repository_github_http_url']
```

Comment:
Ah, this explains some of the `html_url` from `.taskcluster.yml`. Oups, mea culpa :)